src/qt/paymentrequest.proto

```diff
@@ -1,22 +1,23 @@
 //
-// Simple Bitcoin Payment Protocol messages
+// Simple Dogecoin Payment Protocol messages
+// Derived fromthe Bitcoin Payment Protocol
 //
 // Use fields 100+ for extensions;
-// to avoid conflicts, register extensions at:
-// https://en.bitcoin.it/wiki/Payment_Request
+// to avoid conflicts, register extensions via pull-req at:
+// https://github.com/dogecoin/dips
 //
 
 package payments;
-option java_package = "org.bitcoin.protocols.payments";
+option java_package = "com.dogecoin.protocols.payments";
```

Comment:
Interesting, I think I never ran protoc on this to actually generate new Protos for dogecoinj. Might be that it won't even be able to generate/use paymentrequests from this code. Should check that for either dogecoinj or altcoinj. Not a merge blocker here tho.